app/controllers/social_networking/nudges_controller.rb

```diff
@@ -41,20 +39,14 @@ def notify
       when "email"
         send_notify_email(@nudge, message_body)
       when "sms"
-        if recipient.phone_number && !recipient.phone_number.blank?
-          send_sms(recipient, message_body)
-        end
+        send_sms(recipient, message_body) if recipient.phone_number.present?
       when "phone"
-        if recipient.phone_number && !recipient.phone_number.blank?
-          send_sms(recipient, message_body)
-        end
+        send_sms(recipient, message_body) if recipient.phone_number.present?
       else
         logger.error "ERROR: contact preference is not set for \
```

Comment:
This should notify someone if it actually matters, otherwise we should ignore it.